bridgeservice/bridge.go

```diff
@@ -85,6 +86,17 @@ func New(
 	meter := otel.Meter(meterName)
 	cfg.Logger.Infof("starting bridge service (network id=%d)", cfg.NetworkID)
 
+	// The GIN_MODE environment variable controls the mode of the Gin framework.
+	// Valid values are "debug", "release", and "test". If an invalid value is provided,
+	// the mode defaults to "release" for safety and performance.
+	ginMode := os.Getenv("GIN_MODE")
+	switch ginMode {
+	case gin.DebugMode, gin.ReleaseMode, gin.TestMode:
+		gin.SetMode(ginMode)
+	default:
+		gin.SetMode(gin.ReleaseMode) // fallback to release mode
```

Comment:
Should we log this situation at least?

Reply:
Makes sense... I'll add it.

Reply:
Fixed https://github.com/agglayer/aggkit/pull/515/commits/f1e089f0647b96a903c9c4d3adf3381063652b0c